02-express-tutorial/11-methods.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express')
+const app = express()
+let {people} = require('./data')
+
+//static assets
+app.use(express.static('./methods-public'))
+//parse form data
+app.use(express.urlencoded({ extended: false }))
+// parse json
+app.use(express.json())
+
+app.get('/api/people', (req, res) => {
+    res.status(200).json({ success: true, data: people })
+  })
+  
+  app.post('/api/people', (req, res) => {
+    const { name } = req.body
+    if (!name) {
+      return res
+        .status(400)
+        .json({ success: false, msg: 'please provide name value' })
+    }
+    res.status(201).json({ success: true, person: name })
+  })
+
+app.post('/api/postman/people', (req, res) => {
+    const {name} = req.body
+    if (!name) {
+        return res
+        .status(400)
+        .json({success: false, msg: 'please provide name value'})   
+    }
+    res.status(201).json({ success: true, data: [...people, name] })
+})
+
+app.post('/login', (req, res) => {
+    const { name } = req.body
```

Comment:
Formatting nitpick: be consistent about whether or not you put spaces inside the curly braces.